tools/src/wyvern/tools/tests/EffectSystemTests.java

```diff
@@ -384,9 +384,85 @@ public void testNetwork19() throws ParseException {
         TestUtil.doTestScriptModularly(PATH, "effects.testNetwork19", Util.stringType(), new StringLiteral("Network19 with effects"));
     }
 
+    @Test
+    public void accepted2() throws ParseException {
```

Comment:
This test name seems to be inconsistent as there is no test called `accepted` or `accepted1`.